test/components/rankers/test_sentence_transformers_diversity.py

```diff
@@ -607,3 +607,109 @@ def test_run_real_world_use_case(self, similarity):
 
         # Check the order of ranked documents by comparing the content of the ranked documents
         assert result_content == expected_content
+
+    @pytest.mark.integration
```

Comment:
just an explanation: the coverage is decreased because it only takes unit tests into account
https://github.com/deepset-ai/haystack/blob/14895f6573456c6e63852896a856fd87c3624b89/.github/workflows/tests.yml#L122